### ISSUE TITLE
bug on Ubuntu 18.04: Error delivering events through the bus

### ISSUE THREAD

**Issue by PDiracDelta** [Nesvilab/FragPipe#235]
When I click 'Run', it shows the following error. I tried to do an open, nonspecific search, but I'm guessing my settings will not be related to something like "Sticky note". I tried again with other search settings; the same error occurs. Perhaps this is functionality that does not have to crash the program in case of failure?
MSFragger v3.0
```
10:16:54 ERROR: Error delivering events through the bus
java.util.NoSuchElementException: Sticky note not on the bus: com.dmtavt.fragpipe.messages.NoteConfigPhilosopher
	at com.dmtavt.fragpipe.Fragpipe.getStickyStrict(Fragpipe.java:681)
	at com.dmtavt.fragpipe.FragpipeRun.configureTaskGraph(FragpipeRun.java:603)
	at com.dmtavt.fragpipe.FragpipeRun.run(FragpipeRun.java:204)
	at com.dmtavt.fragpipe.tabs.TabRun.on(TabRun.java:153)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:510)
	at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:504)
	at org.greenrobot.eventbus.AsyncPoster.run(AsyncPoster.java:46)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by fcyu** [comment]
It looks like a GUI-related exception specific to your operating system. I am afraid that we cannot fix it. Can you try to upgrade/troubleshoot your OS, or try FragPipe with another operating system, such as Windows?

Best,

Fengchao

**Comment by st-pf** [comment]
Hello. 
I run into the same initial Error warning, although different error descirptions downstream when I run my FragPipe (v17.1) on Windows (see log file below). 

It´s the first time I am trying to set up FragPipe on this computer, do you have an idea what I could try/have to change?

Best wishes,
Stef

[log_2023-01-30_15-39-48.zip](https://github.com/Nesvilab/FragPipe/files/10536789/log_2023-01-30_15-39-48.zip)

**Comment by fcyu** [comment]
Hi Stef,

Could you 1) upgrade FragPipe to the latest version; 2) delete the temp folder in `~/.config/FragPipe` ?

Thanks,

Fengchao

**Comment by st-pf** [comment]
Hey Fengchao, 
thank you for the fast reply.
I installed the v17.1 to be able to run FragPipe without the Java Runtime Environment 1.9
Is the v19.1 really necessary for my error?
When I cleared the FragPipe folder, I still got an error message, even though it looks different now (log in zip folder attached)... What else could I try?
Best wishes and thanks again
Stefan
[log_2023-01-30_16-26-59.zip](https://github.com/Nesvilab/FragPipe/files/10537195/log_2023-01-30_16-26-59.zip)

**Comment by fcyu** [comment]
> Is the v19.1 really necessary for my error?

I don't know since I couldn't reproduce your error. But in case the bug was fixed in the past years (version 17.0 was released in Oct, 2021), you'd better check if the issue is still in the latest version.

> I installed the v17.1 to be able to run FragPipe without the Java Runtime Environment 1.9

Since you are using Windows, you can download the one with `-jre-` in the file name: https://github.com/Nesvilab/FragPipe/releases/tag/19.1 . Please read the release note to find out the difference.

Best,

Fengchao
